cmd/labs/github/releases.go

```diff
@@ -0,0 +1,61 @@
+package github
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/databricks/cli/cmd/labs/localcache"
+	"github.com/databricks/cli/libs/log"
+)
+
+const cacheTTL = 1 * time.Hour
+
+// NewReleaseCache creates a release cache for a repository in the GitHub org.
+// Caller has to provide different cache directories for different repositories.
```

Comment:
Comment seems out of date, now that you pass the pattern to the `localcache`.

Reply:
it's not out of date and is by design.